Makefile

```diff
@@ -14,8 +29,27 @@ build:
 test:
 	go test -v ./...
 
-build-docker-image:
-	docker build -t $(DOCKER_IMAGE_NAME) -f _assets/Dockerfile .
+docker-image:
+	cd _assets/docker && $(MAKE) push
 
 build-platforms:
-	xgo -image $(DOCKER_IMAGE_NAME) --dest $(GOBIN) --targets=linux/amd64,windows/amd64,darwin/amd64 .
+	xgo -image $(IMAGE_NAME) -dest $(GOBIN) -targets=$(XGO_TARGETS) .
```

Comment:
I would change it to `xgo -out keycard ....` so that the executable name is keycard.
it will actually be keycard-os-etc-etc but I think it's fine

Reply:
Fixed.